autoload/lsp/ui/vim.vim

```diff
@@ -435,6 +435,16 @@ function! s:handle_location(ctx, server, type, data) abort "ctx = {counter, list
         if empty(a:ctx['list'])
             call lsp#utils#error('No ' . a:type .' found')
         else
+            " Tagstack supported, added after an idea found here:
+            " https://github.com/prabirshrestha/vim-lsp/issues/262
```

Comment:
Can we remove this comment. git history should allow us to refer to the PR and link to the bug.

Reply:
As you wish.